orbit/pkg/update/flag_runner.go

```diff
@@ -199,12 +200,29 @@ func (r *ExtensionRunner) Run(config *fleet.OrbitConfig) error {
 			return fmt.Errorf("unable to lookup metadata for target: %s, %w", targetName, err)
 		}
 
+		if !r.updateRunner.HasLocalHash(targetName) {
+			log.Info().Msgf("extension %q not present locally; scheduling update", extensionName)
+			extensionsMissing = true
+		}
+
 		sb.WriteString(path + "\n")
 	}
 	if err := os.WriteFile(extensionAutoLoadFile, []byte(sb.String()), constant.DefaultFileMode); err != nil {
 		return fmt.Errorf("error writing extensions autoload file: %w", err)
 	}
 
+	if extensionsMissing {
+		didUpdate, err := r.updateRunner.UpdateAction()
```

Comment:
`UpdateAction` is called every 15 minutes by default (can be customized with `--update-interval=10s`).
So the issue should fix after 15 minutes AFAICS.

Let me know if it makes sense.